### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ZakGriffith**
Update some sentences with clearer wording and fix a typo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,9 +40,9 @@ yarn install
 You'll have three terminals up for:
 
 ```bash
-yarn start   (Next app frontend)
 yarn chain   (hardhat backend)
 yarn deploy  (to compile, deploy, and publish your contracts to the frontend)
+yarn start   (Next app frontend)
```
Comment (rin-st) [inline README.md:45]: Hey! Check `base-challenge-template` branch. This block was changed to this: 


> in the same terminal, start your local network (a blockchain emulator in your computer):

```sh
yarn chain
```

> in a second terminal window, 🛰 deploy your contract (locally):

```sh
cd <challenge_folder_name>
yarn deploy
```

> in a third terminal window, start your 📱 frontend:

```sh
cd <challenge_folder_name>
yarn start
```

📱 Open http://localhost:3000 to see the app.

Comment (ZakGriffith) [reply README.md:45]: I think the question is if they have already learned that from Challenge-0.  If not we can be as verbose as the base template instructions, but if they know it already we can condense it down.  Thoughts?

Comment (carletex) [reply README.md:45]: I think it's not a big deal either way.

I'd personally choose to be as verbose as the base template:
- People might not do the challenges in order (we are also now pointing to specific challenges in the new BG website)
- Consistency

Comment (ZakGriffith) [reply README.md]: Commit #3 adds the verbose instructions, with renumbered checkpoints.

File: README.md [hunk 2]
```diff
@@ -113,7 +120,7 @@ If the balance is less than the `threshold`, you want to set a `openForWithdraw`
 
 ⏳ _"Time Left"_ will only update if a transaction occurs. You can see the time update by getting funds from the faucet button in navbar just to trigger a new block.
 
-If the `Staker UI` tab _"Time Left"_ does not update correctly when a transaction occurs, try switching to other tab and come back to it.
+If the `Staker UI` tab _"Time Left"_ does not update correctly when a transaction occurs, try switching to another tab and come back.
```
Comment (Pabl0cks) [inline README.md:123]: Maybe we can get rid of this sentence, I think is working fine after PR #25

File: README.md [hunk 3]
```diff
@@ -127,12 +134,12 @@ Your `Staker UI` tab should be almost done and working at this point.
 #### 🥅 Goals
 
 - [ ] Can you see `timeLeft` counting down in the `Staker UI` tab when you trigger a transaction with the faucet button?
-- [ ] If you `stake()` enough ETH before the `deadline`, does it call `complete()`?
-- [ ] If you don't `stake()` enough can you `withdraw()` your funds?
+- [ ] If enough Eth is staked by the deadline, does your `execute()` function correctly call `complete()` and stake the Eth?
```
Comment (Pabl0cks) [inline README.md:137]: Sorry about this one, very nitpicky, don't hate me! Think we are using ETH in many places of the Readme, I don't have any special preference, but I think we should use the same term in all the document, either ETH or Eth

Comment (ZakGriffith) [reply README.md]: Good catch.  It has been updated to ETH to match the others.

File: README.md (hunk 2 above)
Comment (ZakGriffith) [reply README.md]: Agreed.  It has been removed.

**Comment by Pabl0cks** [comment]
> ### Checkpoint 4: 🚢 Ship it 🚁
> ### Checkpoint 5: 🎚 Frontend 🧘‍♀️

I was thinking about giving more consistency to those Checkpoint titles across the Challenges.

In the base template and Challenges 0 and 2 we used the term Deploy for the contract and Ship for the frontend. I've just commited a change in the Challenge 2 PR to rename them a bit to be more clear about, do you think would be a good idea to give them a similar name?

- Checkpoint 4: 💾 Deploy your contracts! 🛰
- Checkpoint 5: 🚢 Ship your frontend! 🚁

If you don't like the terms we can adapt them, but I think would be nice to have the same names in all the Challenges.

**Comment by ZakGriffith** [comment]
Yes, 100% agree that getting the Checkpoint names and numbers as consistent as we can will be beneficial.  Noticed that the base template has the yarn chain, yarn deploy, and yarn start in the first (0) checkpoint, while the next challenges have some of that as in the next checkpoint (1).  I like using 'Checkpoint 0 Environment' for the entire setup process (clone, install, chain, etc...)  Thoughts on that?

**Comment by Pabl0cks** [comment]
I've just created PRs #30 #31 and #32 for the Checkpoint titles homogenization

**Review by Pabl0cks — APPROVED** [review]
Resolved a small conflict, I see everything OK

**Pull request merged**